activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -1,4 +1,7 @@
 module ActiveSupport
+  # These load hooks are how Rails communicates that a component of Rails like
+  # ActionView has fully loaded and that component can now be extended.
```

Comment:
I'm trying to refer to the library in Rails called `ActionView`, do you still think adding a space makes more sense. How do we refer to it in other parts of our docs?